dcpy/connectors/github.py

```diff
@@ -36,6 +39,19 @@ def is_running(self) -> bool:
         return self.status in ["queued", "in_progress"]
 
 
+def download_repo(
```

Comment:
Should we just call this `clone`, to mirror GH terminology? Also `f"https://github.com/NYCPlanning/` seems ripe for moving to dcpy.config.